pandas/tests/series/test_analytics.py

```diff
@@ -1265,12 +1273,19 @@ def test_idxmax(self):
         result = s.idxmin()
         assert result == 1.1
 
-    def test_numpy_argmax(self):
-
-        # argmax is aliased to idxmax
+    def test_numpy_argmax_deprecated(self):
         data = np.random.randint(0, 11, size=10)
-        result = np.argmax(Series(data))
-        assert result == np.argmax(data)
+
+        with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
+            # The deprecation of Series.argmax also causes a deprecation warning
+            # when calling np.argmax
```

Comment:
Same comment as <a href="https://github.com/pandas-dev/pandas/pull/16955#discussion_r128038835">above</a> for `argmin`.